content/blog/2019/06/2019-06-14-performance-testing-jenkins.adoc

```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title: "Micro-benchmarking Framework for Jenkins Plugins"
+tags:
+- jmh
+- plugins
+- benchmark
+- performance
+- developer
+- gsoc
+- gsoc2019
+author: abhyudayasharma
+---
+
+= Micro-benchmarking Framework for Jenkins Plugins
+
+I have been working on improving the performance of the Role Strategy Plugin as a part of my link:https://summerofcode.withgoogle.com/projects/#6612887823450112[Google Summer of Code project].
```

Comment:
Fine with me, but maybe https://jenkins.io/projects/gsoc/2019/role-strategy-performance/ is a better link

Reply:
In such case `link:/projects/gsoc/2019/role-strategy-performance/[Google Summer of Code project]` could be used